### PR TITLE
[IA-1648] Create separate metrics for AoU and Terra cluster transitions

### PR DESCRIPTION
**Pull request by akarukappadath**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rtitle — APPROVED** [review]
:shipit:

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterUIConfig.scala [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+case class ClusterUIConfig(
```
Comment (Qi77Qi) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterUIConfig.scala]: can you add `final`

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala [hunk 2]
```diff
@@ -148,6 +148,25 @@ object ClusterContainerServiceType extends Enum[ClusterContainerServiceType] {
...
+  final case object Terra extends ClusterUI {
+    override def asString: String = "Terra"
+  }
+  final case object AoU extends ClusterUI {
+    override def asString: String = "AoU"
+  }
+  final case object Other extends ClusterUI {
+    override def asString: String = "Other"
+  }
+  def getClusterUI(cluster: Cluster): ClusterUI =
```
Comment (Qi77Qi) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala]: can you pass in `labels` instead of the whole `Cluster`?

**Review by Qi77Qi — APPROVED** [review]
👍  2 minor comments

**Comment by akarukappadath** [comment]
jenkins retest

**Pull request merged**